front/app/components/admin/PostManager/components/FilterSidebar/index.tsx

```diff
@@ -125,13 +119,10 @@ const FilterSidebar = ({
         {title}
         {selectionSign}&nbsp;
         {active ? (
-          <Popup
-            content={formatMessage(tooltipMessage)}
-            trigger={
-              <button>
-                <InfoIcon name="info-solid" />
-              </button>
-            }
+          <IconTooltip
```

Comment:
The text is centered now (which is not the case on master): 
<img width="327" alt="image" src="https://github.com/user-attachments/assets/3607e1ee-47d5-478b-bc7b-18f51b0e3007" />

Reply:
Pushed a fix for this.